py_irt/models/one_param_logistic.py

```diff
@@ -211,6 +213,16 @@ def fit_MCMC(self, models, items, responses, num_epochs):
         print(theta_sum)
         print(b_sum)
 
+    def predict(self, models, items, params_from_file=None):
```

Comment:
I think we should be consistent and use `subjects` here

Reply:
Agree, updated throughout.